buildroot/bin/build_marlin_pio

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-platformio run --project-dir $@ --silent
+platformio run --project-dir $@
```

Comment:
I'm not sure this is needed for Travis, all warnings and errors are output with the silent flag and if we leave the file progress enabled when we start building for more platforms (or if there are a few errors) the log will be too long for Travis to display completely. Why do you prefer having the file list output?

Reply:
You are correct. I didn't realize that the warnings would still be printed with the --silent flag. In any case, I'm going to revert this change before I call this PR ready.